cub/test/test_util.h

```diff
@@ -45,6 +45,9 @@
 
 #include <thrust/iterator/discard_iterator.h>
 
+#include <cuda/std/__algorithm/max.h>
+#include <cuda/std/__algorithm/min.h>
```

Comment:
I believe we settled on including the public headers in tests

Reply:
I followed @fbusato's suggestions, but I can rewrite the includes again.

We don't have `<cuda/std/algorithm>`. I assume you mean `<cuda/std/__algorithm_>`. I didn't noticed that one before.